tests/__mocks__/results/data-purge-result.json

```diff
@@ -52,52 +52,52 @@
        <a href=\"https://github.com/Meniole/bot/issues/3#issuecomment-2076628605\" data-hovercard-type=\"issue\" data-hovercard-url=\"/Meniole/bot/issues/3/hovercard\">``` heyo ```</a>               </h6>             </td>             <td>             <details>               <summary>                 0.4               </summary>               <pre class=\"notranslate\">p:   count: 1   score: 1 code:   count: 1   score: 1 </pre>              </details>             </td>             <td>-</td>             <td>-</td>           </tr>           <tr>             <td>               <h6 dir=\"auto\">                 <a href=\"https://github.com/Meniole/bot/issues/3#issuecomment-2076632071\" data-hovercard-type=\"issue\" data-hovercard-url=\"/Meniole/bot/issues/3/hovercard\">gr</a>               </h6>             </td>             <td>             <details>               <summary>                 0.2               </summary>               <pre class=\"notranslate\">p:   count: 1   score: 1 </pre>              </details>             </td>             <td>-</td>             <td>-</td>           </tr>           <tr>             <td>               <h6 dir=\"auto\">                 <a href=\"https://github.com/Meniole/bot/issues/3#issuecomment-2076960228\" data-hovercard-type=\"issue\" data-hovercard-url=\"/Meniole/bot/issues/3/hovercard\">te</a>               </h6>             </td>             <td>             <details>               <summary>                 0.2               </summary>               <pre class=\"notranslate\">p:   count: 1   score: 1 </pre>              </details>             </td>             <td>-</td>             <td>-</td>           </tr>           <tr>             <td>               <h6 dir=\"auto\">                 <a href=\"https://github.com/Meniole/bot/issues/3#issuecomment-2076964178\" data-hovercard-type=\"issue\" data-hovercard-url=\"/Meniole/bot/issues/3/hovercard\">fwe</a>               </h6>             </td>             <td>             <details>               <summary>                 0.2               </summary>               <pre class=\"notranslate\">p:   count: 1   score: 1 </pre>              </details>             </td>             <td>-</td>             <td>-</td>           </tr>           <tr>             <td>               <h6 dir=\"auto\">                 <a href=\"https://github.com/Meniole/bot/issues/3#issuecomment-2076972544\" data-hovercard-type=\"issue\" data-hovercard-url=\"/Meniole/bot/issues/3/hovercard\">te</a>               </h6>             </td>             <td>             <details>               <summary>                 0.2               </summary>               <pre class=\"notranslate\">p:   count: 1   score: 1 </pre>              </details>             </td>             <td>0.9</td>             <td>0.18</td>           </tr>         </tbody>       </table>     </details>       </td>     </tr>   </tbody> </table> </task-lists> [😂](https://emojipedia.org/face-with-tears-of-joy) - elem 1 - elem 2",
```

Comment:
I really never liked how you leave in all this whitespace. In the future it would be nice to fix your renderer.

Reply:
I think it should be just compressed without spaces so the payload is as short as possible. I can do that on a separate pull-request: https://github.com/ubiquibot/conversation-rewards/issues/69